### ISSUE TITLE
sql: Need statement to clearly determine if enterprise features are available

### ISSUE THREAD

**Issue by mrtracy** [cockroachdb/cockroach#21675]
There is currently no clean statement available from a SQL prompt that will allow a user to determine with certainty if enterprise features are enabled.  This has consequences for developers building third-party tools to work with CockroachDB.

Our current best workaround is `SHOW CLUSTER SETTING enterprise.license`, which will return if an enterprise license has been set, but cannot tell you if the license is currently valid or not.

Some early suggestions for the syntax on the clearer function:

+ "`show cluster setting enterprise.is_active` or something and have that cluster setting be un-settable"
+ `select valid_license from crdb_internal.enterprise_status`

**Comment by knz** [comment]
@dt what do you think is the easiest we can do about this which is the most effective? 
I would personally recommend a vtable `crdb_internal.enterprise_features` with two columns: `feature` and `enabled` where the enabled bit (boolean) would be set if there's a license for that feature. WDYT?

**Comment by dt** [comment]
I'm not sure I want to expose an enumeration of enterprise features -- that implies we'd need some sort of central registry of features. Currently we don't have per-feature flags in licenses, just license *types* that the individual features examine to determine if they are enabled or not. We took that approach since we didn't want to have to re-issue licenses with new flags when we added new features.

Do we actually need per-feature status in SQL? If it is just for turning UI features on and off, I'd rather  return just the license type and let the UI figure out what to do with that information.
